modules/gitops-dex-role-mappings.adoc

```diff
@@ -0,0 +1,19 @@
+// Module is included in the following assemblies:
+//
+// * configuring-sso-for-argo-cd-on-openshift
+
+[id="dex-role-mappings_{context}"]
+= Mapping users to specific roles
+
+You can set the role of a specific user as `role:admin` on SSO through OpenShift. The user should be in a group in which the `cluster-admin` role is added. Argo CD role will not map if the user has a direct `ClusterRoleBinding` role. A quick fix is to:
```

Comment:
Consider the following rephrase to the last two sentences:

`Argo CD cannot map to users if they have a direct `ClusterRolfeBinding` role. You can manually change users' roles.` 

Procedure 

<Steps that you have> 

Note, the SG wants us to avoid self referential language, like "use this procedure to BLANK." Instead, utilize the introduction to set up the procedure so the user knows what the procedure accomplishes.